src/message.py

```diff
@@ -1,34 +1,34 @@
 class Message:
-    def display_welcome_message(self):
-        print("Welcome to Tic Tac Toe")
+    def welcome_message(self):
+        return "Welcome to Tic Tac Toe"
 
-    def display_game_over_message(self):
-        print("Game Over!")
+    def rules(self):
+        rules = """\nPlay this game by taking turns marking the board. \nWhen prompted, type your a number between 1 and 9 and press enter. \nIf that spot is taken, the computer will prompt you for a different spot. \nThe first player who gets three of their marks in a row wins! \nIf the board is full and neither player has three in a row, it is a draw and the game is over. \nAt the end of every game, you will have the option to play again or to exit.\n"""
```

Comment:
non-blocking; Does Python offer a way to do a multi-line string?  The length of this one line is pretty long, and not very developer friendly.

Reply:
https://www.w3schools.com/python/gloss_python_multi_line_strings.asp --> fixed formatting